go/glide.yaml

```diff
@@ -25,7 +25,6 @@ import:
   subpackages:
   - lib
 - package: github.com/Sirupsen/logrus
-  version: 4b6ea7319e214d98c938f12692336f7ca9348d6b
```

Comment:
I think logrus has semver tags now, should we pin to the current version (or patch updates thereof)?